README.md

```diff
@@ -213,6 +213,17 @@ Porting from Lit-LLaMA in progress 👷
 
 &nbsp;
 
+## XLA
+
+Lightning AI has partnered with Google to add first-class support for Cloud TPUs in Lightning’s frameworks and Lit-GPT,
```

Comment:
(Optional) we can add a link for "Cloud TPUs"? Something like this: https://cloud.google.com/tpu